app/help/ru/components/AccountMembership.md

```diff
@@ -1,10 +1,10 @@
-[# пожизненно]
+[# lifetime]
```

Comment:
Why did you put english titles here now?

Reply:
reverted ru

Reply:
There are two more cases like that in the PR